framework/console/controllers/MigrateController.php

```diff
@@ -166,9 +166,7 @@ public function optionAliases()
     public function beforeAction($action)
     {
         if (parent::beforeAction($action)) {
-            if ($action->id !== 'create') {
-                $this->db = Instance::ensure($this->db, Connection::className());
-            }
+            $this->db = Instance::ensure($this->db, Connection::className());
```

Comment:
I'd leave it as is here and duplicate the call in `create` action so it won't trigger errors if it doesn't need database.